st2common/st2common/util/jsonify.py

```diff
@@ -73,8 +76,8 @@ def json_loads(obj, keys=None):
     for key in keys:
         try:
             obj[key] = json.loads(obj[key])
-        except:
-            pass
+        except Exception as e:
+            LOG.exception('Criteria pattern not valid JSON : "%s"', e)
```

Comment:
Log exception already includes the exception object so there's no need to reprint `e`.